dijet/production/jet_assignment.py

```diff
@@ -91,11 +91,18 @@ def jet_assignment(self: Producer, events: ak.Array, **kwargs) -> ak.Array:
     reference_jet = ak.firsts(jets[ak.singletons(ref_index)])
 
     # write out probe and reference jet fields
-    fields = ("pt", "eta", "phi", "mass")
+    fields = ("pt", "eta", "phi", "mass", "genJetIdx") if self.dataset_inst.is_mc else ("pt", "eta", "phi", "mass")
```

Comment:
Line seems a bit long. Does the linter complain? In any case, it could be split up into separate lines. Or even better, the tuple could be extended in a conditional block to avoid repeating the common fields between data/MC:
```suggestion
    fields = ("pt", "eta", "phi", "mass")
    if self.dataset_inst.is_mc:
        fields += ("genJetIdx",)
```